docker/app/run-celery.sh

```diff
@@ -5,3 +5,4 @@
   worker \
   --concurrency="${QUEUE_WORKERS}" \
   --loglevel="${LOKOLE_LOG_LEVEL}"
+  --Q="${STORE_WRITTEN_QUEUE_NAME}","${SEND_QUEUE_NAME}","${STORE_INBOUND_QUEUE_NAME}"
```

Comment:
As per the comment above, I'd switch to something like this:

```suggestion
  --Q="${CELERY_QUEUE_NAMES}"
```

We can then set the value of `CELERY_QUEUE_NAMES` to `inbound_store,written_store,send` in `docker-compose.yml` and `worker-deployment.yaml`.

The nice property of this approach is that we can re-use the same configuration to run multiple workers for monitoring the queues independently by creating another copy of the worker container with `CELERY_QUEUE_NAMES` set to only `inbound_store`, `written_store` or `send`.

Thoughts?

Reply:
Ok, got it! I think this is a much better idea. Thanks for the insight!